### PR TITLE
adds styles header and hero

### PR DESCRIPTION
**Pull request by Yuriy-St**

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+	--main-bg-color: #ffffff;
+}
+
+body {
+	font-family: 'Open Sans', sans-serif;
+	color: var(--main-text-color);
+	font-size: 18px;
+	background-color: var(--main-bg-color);
+}
+
```
Comment (TetraQuadra) [inline css/styles.css:14]: Comment

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+h2,
+h3,
+h4,
+h5,
+h6,
+p {
+	margin: 0;
+}
+
+.list {
```
Comment (TetraQuadra) [inline css/styles.css:26]: Edit style

**Review by TetraQuadra — COMMENTED** [review]
Fix global styles

File: css/styles.css (hunk 1 above)
Comment (Yuriy-St) [reply css/styles.css:14]: Fixed

File: css/styles.css (hunk 2 above)
Comment (Yuriy-St) [reply css/styles.css:26]: Done

**Review by TetraQuadra — APPROVED** [review]

**Review by TetraQuadra — APPROVED** [review]

**Review by TetraQuadra — APPROVED** [review]

**Pull request merged**